doc/plutip-testing.md

```diff
@@ -193,3 +194,34 @@ let
 Although stake keys serve no real purpose in plutip context, they allow to use base addresses, and thus allow to have the same code for plutip testing, in-browser tests and production.
 
 Note that CTL re-distributes tADA from payment key-only ("enterprise") addresses to base addresses, which requires a few transactions before the test can be run. Plutip can currently handle only enterprise addreses (see [this issue](https://github.com/mlabs-haskell/plutip/issues/103)).
+
+## CTL/Plutip utilities for testing
+
+`Contract.Test.Utils` module provides a DSL for assertions that accumulate error messages, instead of exiting early after the first failure.
+
+The interpreter is `withAssertions` function, that accepts two arrays for two kinds of assertions:
```

Comment:
I think the statement that `withAssertions` accepts **two arrays** is a bit inaccurate, maybe replace it with something like:
```suggestion
The interpreter is `withAssertions` function, that executes specified assertions (which can be combined using `Array`s and `Tuple`s):
```